apps/passport-server/src/routing/routes/telegramRoutes.ts

```diff
@@ -275,4 +276,16 @@ export function initTelegramRoutes(
       res.status(500).send(errorHtmlWithDetails(e as Error));
     }
   });
+
+  app.get("/telegram/bot", async (req: Request, res: Response) => {
+    const ping = telegramService?.ping();
+    if (ping) return res.status(200).send(`Auth bot is running`);
```

Comment:
We can remove this check and just have a re-start route, but prob don't want to expose it publicly